src/SignalR/server/Core/src/Internal/SignalRServerActivitySource.cs

```diff
@@ -8,7 +8,10 @@ namespace Microsoft.AspNetCore.SignalR.Internal;
 // Internal for now so we don't need API review.
 // Just a wrapper for the ActivitySource
 // don't want to put ActivitySource directly in DI as hosting already does that and it could get overwritten.
-internal sealed class SignalRActivitySource
+internal sealed class SignalRServerActivitySource
 {
-    public ActivitySource ActivitySource { get; } = new ActivitySource("Microsoft.AspNetCore.SignalR.Server");
+    internal const string Name = "Microsoft.AspNetCore.SignalR.Server";
+    internal const string InvocationIn = $"{Name}.InvocationIn";
```

Comment:
Is this a convention or just a name we've landed on?
Would we have an "InvocationOut" in the future, for example if the server calls a client method?